guides/deployment/releases.md

```diff
@@ -189,26 +189,41 @@ ENV MIX_ENV=prod
 
 # install mix dependencies
 COPY mix.exs mix.lock ./
-COPY config config
-RUN mix do deps.get --only $MIX_ENV, deps.compile
+RUN mix deps.get --only $MIX_ENV
+RUN mkdir config
+# if our compile-time config changes, we may need to
+# recompile dependencies
```

Comment:
Is this something done automatically for us? Or would we need to trigger it manually?

Reply:
Clarifying.